stdlib/public/stubs/UnicodeNormalization.cpp

```diff
@@ -214,7 +237,8 @@ void *swift::_swift_stdlib_unicodeCollationIterator_create(
       ucol_openElements(GetRootCollator(), reinterpret_cast<const UChar *>(Str),
                         Length, &ErrorCode);
   if (U_FAILURE(ErrorCode)) {
-    swift::crash("_swift_stdlib_unicodeCollationIterator_create: ucol_openElements() failed.");
+    swift::crash("_swift_stdlib_unicodeCollationIterator_create: "
```

Comment:
All this collation code can be removed, right?

Reply:
String hashing still uses some of these functions on Linux